sql/scheduledlogging: TestCaptureIndexUsageStats failed

sql/scheduledlogging.TestCaptureIndexUsageStats [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/10005244?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/10005244?buildTab=artifacts#/) on master @ [7a8be52cc09dd6779b02a54bf7ce9a290cd135c8](https://github.com/cockroachdb/cockroach/commits/7a8be52cc09dd6779b02a54bf7ce9a290cd135c8):


```
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881908000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881907294,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":2,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.646766 +0000 UTC","SchemaName":"public"} n1 16 true TELEMETRY 326 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881882000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881881300,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":1,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.378305 +0000 UTC","SchemaName":"public"} n1 15 true TELEMETRY 334 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881864000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881862896,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":2,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.391717 +0000 UTC","SchemaName":"public"} n1 14 true TELEMETRY 323 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881845000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881845074,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":1,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.373061 +0000 UTC","SchemaName":"public"} n1 13 true TELEMETRY 331 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881829000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881828639,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":2,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"IdX✏","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.890136 +0000 UTC","SchemaName":"public"} n1 12 true TELEMETRY 329 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881801000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881800905,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":1,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"sPe-CiAl✔_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.382678 +0000 UTC","SchemaName":"public"} n1 11 true TELEMETRY 353 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881779000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881778251,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":2,"DatabaseName":"index","TableName":"index","IndexName":"index","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:16.096715 +0000 UTC","SchemaName":"public"} n1 10 true TELEMETRY 303 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881725000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881721946,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":1,"DatabaseName":"index","TableName":"index","IndexName":"index_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.387427 +0000 UTC","SchemaName":"public"} n1 9 true TELEMETRY 322 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491650000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491649881,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":2,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.646766 +0000 UTC","SchemaName":"public"} n1 8 true TELEMETRY 326 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491640000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491639756,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":1,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.378305 +0000 UTC","SchemaName":"public"} n1 7 true TELEMETRY 334 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491632000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491632231,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":2,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.391717 +0000 UTC","SchemaName":"public"} n1 6 true TELEMETRY 323 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491624000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491624125,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":1,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.373061 +0000 UTC","SchemaName":"public"} n1 5 true TELEMETRY 331 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491615000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491615074,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":2,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"IdX✏","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.890136 +0000 UTC","SchemaName":"public"} n1 4 true TELEMETRY 329 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491604000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491603308,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":1,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"sPe-CiAl✔_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.382678 +0000 UTC","SchemaName":"public"} n1 3 true TELEMETRY 353 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491578000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491573654,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":2,"DatabaseName":"index","TableName":"index","IndexName":"index","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:16.096715 +0000 UTC","SchemaName":"public"} n1 2 true TELEMETRY 303 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664700721386000 2701 util/log/event_log.go 32 {"Timestamp":1683664700721382728,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":1,"DatabaseName":"index","TableName":"index","IndexName":"index_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.387427 +0000 UTC","SchemaName":"public"} n1 1 true TELEMETRY 322 0 0 1}
    captured_index_usage_stats_test.go:324: found index counts: map[IdX✏:2 index:2 index_pkey:2 sPe-CiAl✔_pkey:2 test2_table_letter_idx:2 test2_table_pkey:2 test_table_letter_idx:2 test_table_pkey:2]
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561928000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561927253,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":2,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.646766 +0000 UTC","SchemaName":"public"} n1 24 true TELEMETRY 326 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561899000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561898248,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":1,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.378305 +0000 UTC","SchemaName":"public"} n1 23 true TELEMETRY 334 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561882000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561881640,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":2,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.391717 +0000 UTC","SchemaName":"public"} n1 22 true TELEMETRY 323 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561864000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561863523,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":1,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.373061 +0000 UTC","SchemaName":"public"} n1 21 true TELEMETRY 331 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561839000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561838530,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":2,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"IdX✏","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.890136 +0000 UTC","SchemaName":"public"} n1 20 true TELEMETRY 329 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561815000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561813882,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":1,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"sPe-CiAl✔_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.382678 +0000 UTC","SchemaName":"public"} n1 19 true TELEMETRY 353 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561791000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561790357,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":2,"DatabaseName":"index","TableName":"index","IndexName":"index","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:16.096715 +0000 UTC","SchemaName":"public"} n1 18 true TELEMETRY 303 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664706561738000 2701 util/log/event_log.go 32 {"Timestamp":1683664706561734578,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":1,"DatabaseName":"index","TableName":"index","IndexName":"index_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.387427 +0000 UTC","SchemaName":"public"} n1 17 true TELEMETRY 322 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881908000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881907294,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":2,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.646766 +0000 UTC","SchemaName":"public"} n1 16 true TELEMETRY 326 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881882000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881881300,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":1,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.378305 +0000 UTC","SchemaName":"public"} n1 15 true TELEMETRY 334 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881864000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881862896,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":2,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.391717 +0000 UTC","SchemaName":"public"} n1 14 true TELEMETRY 323 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881845000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881845074,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":1,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.373061 +0000 UTC","SchemaName":"public"} n1 13 true TELEMETRY 331 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881829000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881828639,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":2,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"IdX✏","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.890136 +0000 UTC","SchemaName":"public"} n1 12 true TELEMETRY 329 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881801000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881800905,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":1,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"sPe-CiAl✔_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.382678 +0000 UTC","SchemaName":"public"} n1 11 true TELEMETRY 353 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881779000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881778251,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":2,"DatabaseName":"index","TableName":"index","IndexName":"index","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:16.096715 +0000 UTC","SchemaName":"public"} n1 10 true TELEMETRY 303 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664704881725000 2701 util/log/event_log.go 32 {"Timestamp":1683664704881721946,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":1,"DatabaseName":"index","TableName":"index","IndexName":"index_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.387427 +0000 UTC","SchemaName":"public"} n1 9 true TELEMETRY 322 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491650000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491649881,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":2,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.646766 +0000 UTC","SchemaName":"public"} n1 8 true TELEMETRY 326 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491640000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491639756,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":113,"IndexID":1,"DatabaseName":"test2","TableName":"test2_table","IndexName":"test2_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.378305 +0000 UTC","SchemaName":"public"} n1 7 true TELEMETRY 334 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491632000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491632231,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":2,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_letter_idx","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.391717 +0000 UTC","SchemaName":"public"} n1 6 true TELEMETRY 323 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491624000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491624125,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":112,"IndexID":1,"DatabaseName":"test","TableName":"test_table","IndexName":"test_table_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.373061 +0000 UTC","SchemaName":"public"} n1 5 true TELEMETRY 331 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491615000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491615074,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":2,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"IdX✏","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:14.890136 +0000 UTC","SchemaName":"public"} n1 4 true TELEMETRY 329 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491604000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491603308,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":114,"IndexID":1,"DatabaseName":"\"mIxEd-CaSe\"\"woo☃\"","TableName":"sPe-CiAl✔","IndexName":"sPe-CiAl✔_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.382678 +0000 UTC","SchemaName":"public"} n1 3 true TELEMETRY 353 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664702491578000 2701 util/log/event_log.go 32 {"Timestamp":1683664702491573654,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":2,"DatabaseName":"index","TableName":"index","IndexName":"index","IndexType":"secondary","CreatedAt":"2023-05-09 20:38:16.096715 +0000 UTC","SchemaName":"public"} n1 2 true TELEMETRY 303 0 0 1}
    captured_index_usage_stats_test.go:307: checking entry: {INFO 1683664700721386000 2701 util/log/event_log.go 32 {"Timestamp":1683664700721382728,"EventType":"captured_index_usage_stats","LastRead":"0001-01-01 00:00:00 +0000 UTC","TableID":115,"IndexID":1,"DatabaseName":"index","TableName":"index","IndexName":"index_pkey","IndexType":"primary","IsUnique":true,"CreatedAt":"2023-05-09 20:38:14.387427 +0000 UTC","SchemaName":"public"} n1 1 true TELEMETRY 322 0 0 1}
    captured_index_usage_stats_test.go:324: found index counts: map[IdX✏:3 index:3 index_pkey:3 sPe-CiAl✔_pkey:3 test2_table_letter_idx:3 test2_table_pkey:3 test_table_letter_idx:3 test_table_pkey:3]
    captured_index_usage_stats_test.go:267: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/scheduledlogging/captured_index_usage_stats_test.go:267
        	Error:      	"1s" is not greater than "1.160339s"
        	Test:       	TestCaptureIndexUsageStats
        	Messages:   	1s <= 4.160339s-3s
test logs left over in: /artifacts/tmp/_tmp/69cc8b5ff8b35dcd9569961cf48105ef/logTestCaptureIndexUsageStats2024413349
    panic.go:522: -- test log scope end --
--- FAIL: TestCaptureIndexUsageStats (17.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102732 sql/scheduledlogging: TestCaptureIndexUsageStats failed [C-test-failure O-robot T-cluster-observability branch-release-23.1.0]
</p>
</details>
/cc @cockroachdb/cluster-observability
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCaptureIndexUsageStats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-27768

--- a/pkg/sql/scheduledlogging/captured_index_usage_stats_test.go
+++ b/pkg/sql/scheduledlogging/captured_index_usage_stats_test.go
@@ -80,7 +80,7 @@ func TestCaptureIndexUsageStats(t *testing.T) {
 	stubLoggingDelay := 0 * time.Second
 
 	// timeBuffer is a short time buffer to account for non-determinism in the logging timings.
-	const timeBuffer = 3 * time.Second
+	const timeBuffer = 4 * time.Second
 
 	settings := cluster.MakeTestingClusterSettings()
 	// Configure capture index usage statistics to be disabled. This is to test

